.circleci/config.yml

```diff
@@ -167,47 +166,52 @@ commands:
           key: << parameters.key_prefix >>-{{ .Branch }}
 
   manage_api_cache:
-    description: "workbench/api: save, restore or save_and_restore gradle cache"
+    description: "workbench/api: save or restore gradle cache"
     parameters:
       save:
         type: boolean
         default: false
       restore:
         type: boolean
         default: false
-      key_prefix:
-        type: string
-        default: v1-api-cache-{{ checksum "~/workbench/api/build.gradle" }}
     steps:
       - when:
           condition: << parameters.save >>
           steps:
+            # splitting into multiple distinct caches. Refers to https://circleci.com/docs/2.0/caching/#cache-size
```

Comment:
I splitted one cache to two separate caches. See url for reason.

Reply:
What is the size of the cache? Are we actually nearing 500MB?

Reply:
The cache is best effort. In theory all jobs should succeed regardless of whether they had received an ideal or even relevant cache. gradle wapper properties very rarely changes, so I would probably favor the simplicity of the previous singular cache approach. If we do get stuck with a cache issue, incrementing from "v1" is a nice manual out to have available that you've added.